src/dev/dev_command.ts

```diff
@@ -34,9 +39,28 @@ export async function dev(
 
   if (manifestChanged) await generate(dir, newManifest);
 
-  if (Deno.args.includes("build")) {
+  const manifest = (await import(toFileUrl(join(dir, "fresh.gen.ts")).href))
+    .default as ServerManifest;
```

Comment:
If `fresh.gen.ts` is not present on disk, this will throw. Is it handled anywhere?

Reply:
In this dev script we generate the `fresh.gen.ts` manifest a few lines before this one ourselves. I think it's ok to assume that it will be present here.